ui/src/tasks/components/TaskRow.tsx

```diff
@@ -104,7 +104,7 @@ export class TaskRow extends PureComponent<Props & WithRouterProps> {
     }
 
     return (
-      <Label.Container limitChildCount={4}>
+      <Label.Container limitChildCount={4} resourceName="this Task">
```

Comment:
should `this` be capitalized?

Reply:
it's appended to a different string. Now that you point that out, maybe that should be more clear